docs/releases/overview.md

```diff
@@ -15,11 +15,16 @@ Increments to major release imply API breaking changes or significant new featur
 The second number corresponds with a minor release number within the major release number.
 Increments to the minor release imply bug fixes or small feature additions that should be backwards compatible with code that currently integrates with the major release number.
 
-The latest release can be found at
+The latest stable release can be found at
```

Comment:
does this mean that http://edu.irobot.com/create3-latest-fw will keep pointing at G.3.1 until Humble is beta?

Reply:
Yes, for now. We need to decide what our policy will be for -latest-fw. Maybe we can talk at our next meeting.